_posts/2022-11-09-WildFly27-Final-Released.adoc

```diff
@@ -0,0 +1,216 @@
+
+*Provisioning*
+
+We've added a number of new link:https://docs.wildfly.org/27/Galleon_Guide.html#wildfly_galleon_layers[Galleon layers]:
+
+* link:https://issues.redhat.com/browse/WFLY-13798['embedded-activemq'] for embedded broker messaging.
+* link:https://issues.redhat.com/browse/WFLY-17004['hibernate-search'] to support use of Hibernate Search with the `jpa` layer.
+* link:https://issues.redhat.com/browse/WFLY-16452['mod_cluster'] for mod_cluster subsystem provisioning.
+* link:https://issues.redhat.com/browse/WFLY-16453['singleton-local' and 'singleton-ha'] for different configuration flavors using the `singleton` subsytem.
+
+*Hibernate Search*
+
+We've updated the version of Hibernate Search we integrate up to the 6.1 series. We've also made a couple of other changes in our Hibernate Search support:
+
+* A number of modules formerly only intended for internal access by Hibernate Search have now been link:https://issues.redhat.com/browse/WFLY-16874[marked as public], allowing for direct application use of the APIs exposed by those modules.
+* In WildFly Preview we've link:https://issues.redhat.com/browse/WFLY-16861[added support] for Hibernate Search's experimental feature alllowing  link:https://docs.jboss.org/hibernate/stable/search/reference/en-US/html_single/#architecture-examples-outbox-polling-elasticsearch[coordinated indexing of entities from multiple instances of the same application].
+
+*Major Component Upgrades vs WildFly 26.x*
+
+There have been numerous component upgrades related to WildFly's transition from EE 8 to EE 10; too many to list here. However, besides those EE 10 driven updates, there are a number of other major component updates in WildFly 27:
+
+* Hibernate 6.1 replaces Hibernate 5.3
+* Hibernate Search 6.1 replaces Hibernate Search 5.10
+* Infinispan 14 replaces Infinispan 13
+* JGroups 5.2 replaces JGroups 4.2
+* RESTEasy 6.2 replaces RESTEasy 4.7
+* Weld 5 replaces Weld 3.1
+
+== MicroProfile Support
+
+WildFly's MicroProfile support includes implementations of the following specifications in our "full" (e.g. `standalone-full.xml`) and "default" (e.g `standalone.xml`) configurations as well as our "microprofile" configurations (e.g. `standalone-microprofile.xml`):
+
+[cols=",,",options="header"]
+|=======================================================================
+|MicroProfile Technology |WildFly Full/Default Configurations |WildFly MicroProfile Configuration
+
+|MicroProfile Config 3.0 |X |X
+
+|MicroProfile Fault Tolerance 4.0 |-- |X
+
+|MicroProfile Health 4.0 |-- |X
+
+|MicroProfile JWT Authentication 2.0 |X |X
+
+|MicroProfile Metrics 4.0 (deprecated in WildFly; will be replaced by link:Admin_Guide{outfilesuffix}#Micrometer_Metrics[Micrometer]) |-- |X
+
+|MicroProfile OpenAPI 3.0 |-- |X
+
+|MicroProfile OpenTracing 3.0 (deprecated; use link:Admin_Guide{outfilesuffix}#Observability_Tracing[OpenTelemetry]) |X |X
```

Comment:
This should probably be https://docs.wildfly.org/27/Admin_Guide.html#Observability_Tracing